app/models/user.rb

```diff
@@ -1,5 +1,15 @@
 class User < ApplicationRecord
+  enum role: { user: 0, admin: 1 }
```

Comment:
Let's move the enum below the has_one relations.

We can follow a file format like:
- has_relations
- enums
- validations
- methods